web3-dev/smart-contracts/sc-execution.rst

```diff
@@ -0,0 +1,68 @@
+========================
+Smart contract execution
+========================
+
+Introduction
+============
+
+In Massa you can execute smart contracts not only deploy and call.
+There is different possibilities to execute smart contracts in different contexts.
+This documentation will explain you the different possibilities and how to use them.
+You can choose the one that match the most your usecase.
+
+All of this way to execute smart contracts are available
+in the smart contract sdk `massa-web3 <https://github.com/massalabs/massa-web3>`_.
+
+Differents ways to execute smart contracts
+==========================================
+
+Operation execute smart contract
+--------------------------------
+
+This operation allow you to send a smart contract that will be executed on the blockchain
+(not stored) and can perform all the state changes as a normal smart contract.
+It's often the best way to deploy an other smart contract using the operation datastore of the execute operation.
+Our `deployer <https://github.com/massalabs/massa-sc-toolkit/blob/main/packages/sc-deployer/src/index.ts>`_
+use this operation to deploy smart contracts.
+
+Operation call smart contact
+----------------------------
+
+This operation allow you to call the function of a smart contract that exists
+on the blockchain with specific parameters and coins.
+
+An `example <https://github.com/massalabs/massa-web3#smart-contract-read-and-write-calls>`_ using massa-web3 will
```

Comment:
English can be improved.

Reply:
I have written it again. Tell me if it's better